src/2.6.0/wso2apim.js

```diff
@@ -123,9 +127,9 @@ async function isAPIDeployed(url, accessToken, apiName, apiVersion, apiContext)
   }
 }
 
-async function isCertUploaded(url, accessToken, certAlias) {
+async function isCertUploaded(wso2APIM, accessToken, certAlias) {
   try {
-    url = url + '/' + certAlias;
+    let url = 'https://' + wso2APIM.host + ':' + wso2APIM.port + '/api/am/publisher/' + wso2APIM.versionSlug + '/certificates/' + certAlias;
```

Comment:
Maybe you can use string templates here? 

```javascript
let url = `https://${wso2APIM.host}:${wso2APIM.port}/api/am/publisher/${wso2APIM.versionSlug}/certificates/${certAlias}`
```

Reply:
Neat one. Let me..

Reply:
Resolved.